protocol/src/common/protocol.configuration.ts

```diff
@@ -25,6 +25,7 @@ export namespace ConfigurationRequest {
 	export const type = new ProtocolRequestType<ConfigurationParams, LSPAny[], never, void, void>(method);
 	export type HandlerSignature = RequestHandler<ConfigurationParams, LSPAny[], void>;
 	export type MiddlewareSignature = (params: ConfigurationParams, token: CancellationToken, next: HandlerSignature) => HandlerResult<LSPAny[], void>;
+	export const capabilities: { client: 'workspace.configuration'; server: 'workspace' } = { client: 'workspace.configuration', server: 'workspace' };
```

Comment:
@dbaeumer The spec doesn't mention a server capability for the `workspace/configuration` request so I'm unsure about this. Since this is requested by the server it seems like there's no server capability needed here?